docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Core_Management.adoc

```diff
@@ -132,3 +132,37 @@ Now we can list the last configuration changes :
     }]
 }
 ----
+
+[[unstable-api-annotation]]
+== [Preview Feature] Reporting of API elements annotated with annotations indicating unstable API
+
+Note: This feature only exists at the `preview` stability level or lower.
```

Comment:
Please make part of this a link to https://docs.wildfly.org/32/Admin_Guide.html#Feature_stability_levels